src/main/kotlin/lotto/domain/Lotto.kt

```diff
@@ -1,21 +1,20 @@
-package lotto
+package lotto.domain
 
 class Lotto {
+    val numbers: List<Int>
+
     constructor() {
-        numbers = LOTTO_NUMBER_POOL.shuffled().subList(0, 6)
+        numbers = LOTTO_NUMBER_POOL.shuffled().subList(0, 6).sorted()
```

Comment:
p3; 새롭게 생성된 객체가 생길 때 개행을 한다면 조금 더 이해하기 쉬운 로직이 될 거 같아요.
디미터 법칙을 한 번 읽어보시면 좋을 거 같습니다.

추가로 다음과 같이 take를 사용해 볼 수도 있을 거 같네요. 😊

```suggestion
        numbers = LOTTO_NUMBER_POOL.shuffled()
            .take(n = NUMBER_OF_LOTTO_NUMBER)
            .sorted()
```